src/utils/viewer/viewer.cpp

```diff
@@ -134,8 +135,11 @@ class Viewer : public Mn::Platform::Application {
   esp::gfx::RenderCamera* renderCamera_ = nullptr;
   esp::nav::PathFinder::ptr pathfinder_;
   esp::scene::ObjectControls controls_;
+  // TODO: not needed anymore, will be depricated in next PR.
   Mn::Vector3 previousPosition_;
```

Comment:
remove now and not later? i don't see it used anymore

Reply:
OK, as you wish.
I was planning to fix it in another diff.
This diff is to just fix exactly one problem and let everybody reach a consensus.